2021-09-17 20:10:03,398 INFO    MainThread:25443 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 20:10:03,399 INFO    MainThread:25443 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 20:10:03,399 INFO    MainThread:25443 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/Summarize-Text-Review/wandb/run-20210917_201003-3mwjp3q3/logs/debug.log
2021-09-17 20:10:03,399 INFO    MainThread:25443 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/Summarize-Text-Review/wandb/run-20210917_201003-3mwjp3q3/logs/debug-internal.log
2021-09-17 20:10:03,399 INFO    MainThread:25443 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fce22455310>
2021-09-17 20:10:03,400 INFO    MainThread:25443 [wandb_init.py:init():375] calling init triggers
2021-09-17 20:10:03,400 INFO    MainThread:25443 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-09-17 20:10:03,400 INFO    MainThread:25443 [wandb_init.py:init():424] starting backend
2021-09-17 20:10:03,400 INFO    MainThread:25443 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-17 20:10:03,436 INFO    MainThread:25443 [backend.py:ensure_launched():135] starting backend process...
2021-09-17 20:10:03,475 INFO    MainThread:25443 [backend.py:ensure_launched():140] started backend process with pid: 32100
2021-09-17 20:10:03,476 INFO    MainThread:25443 [wandb_init.py:init():429] backend started and connected
2021-09-17 20:10:03,540 INFO    MainThread:25443 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-17 20:10:03,541 INFO    MainThread:25443 [wandb_init.py:init():477] updated telemetry
2021-09-17 20:10:03,541 INFO    MainThread:25443 [wandb_init.py:init():500] communicating current version
2021-09-17 20:10:04,431 INFO    MainThread:25443 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-17 20:10:04,432 INFO    MainThread:25443 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-17 20:10:05,054 INFO    MainThread:25443 [wandb_init.py:init():540] starting run threads in backend
2021-09-17 20:10:10,060 INFO    MainThread:25443 [wandb_run.py:_console_start():1601] atexit reg
2021-09-17 20:10:10,060 INFO    MainThread:25443 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-17 20:10:10,061 INFO    MainThread:25443 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-17 20:10:10,061 INFO    MainThread:25443 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-17 20:10:10,061 INFO    MainThread:25443 [wandb_init.py:init():565] run started, returning control to user process
2021-09-17 20:11:11,612 INFO    MainThread:25443 [wandb_init.py:_pause_backend():256] pausing backend
2021-09-17 20:11:11,612 INFO    MainThread:25443 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-09-17 20:12:51,008 INFO    MainThread:25443 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-17 20:12:51,009 INFO    MainThread:25443 [wandb_run.py:_restore():1543] restore
2021-09-17 20:12:51,844 INFO    MainThread:25443 [wandb_run.py:_restore():1543] restore
